packages/rum-recorder/src/domain/segmentCollection/segmentCollection.ts

```diff
@@ -145,8 +145,8 @@ export function doStartSegmentCollection(
           flushSegment('max_size')
         }
       },
-      (data) => {
-        send(data, segment.meta)
+      (data, rawSegmentSize) => {
```

Comment:
I see that we are calling this same argument `segmentSize` above, but `rawSegmentSize` here. I think using the same name in all places would be clearer.

Reply:
Indeed, well caught.